notebooks/GPIOs-API/requirements.txt

```
@@ -0,0 +1 @@
+BinhoSupernova==2.2.0
```

Is this dependency strict, or could a newer Supernova version support it?
```suggestion
BinhoSupernova>=2.2.0
```